README.md

```diff
@@ -1,50 +1,204 @@
-# I-Remote-Procedure-Call
+# Cross-Platform Protocol/Glue/Remote-Procedure-Call
 
-`irpc` facilitates cross platform development. Currently, this project
-is experimental, and we are actively working to improve its functionality
-and optimize its architecture. `irpc` should be easy to use, understand,
-modify, and extend. This project has been developed with two problems in mind:
+`irpc` facilitates cross platform development. Currently, this project is experimental, and we are actively working to improve its functionality and optimize its architecture. `irpc` should be easy to use, understand, modify, and extend. This project has been developed with two problems in mind:
 
-- how to easily integrate a library written in Fortran with a library written
-  in Python
-- how to easily integrate Python libraries that are likely to have incompatible
-  dependency requirements
+- how to easily integrate a library written in Fortran with a library written in Python
```

Comment:
It could be mentioned that an isomorphic issue is how to easily integrate a model written in R into the PolicyBrain webapp written in Python.

Reply:
Are you referring to using the `reticulate` package that I mentioned the other day?

Reply:
I wasn't specifically thinking of that, although that's an example of it. I think at the very beginning of this project, we had specifically discussed the possibility for it to be used to plug a model written in an entirely different language into PolicyBrain.

Reply:
Right, that could be an interesting application of this work. I would rather put it like "how to easily run Tax-Calculator from R without worrying about passing data from R to Python via C bindings"